Resources/Private/Partials/ContentElements/Media/Render/Svg.html

```diff
@@ -1,7 +1,7 @@
 <html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
 
 <f:if condition="{file.alternative}">
-  <f:variable name="role" value="{role: 'img'}" />
+  <f:variable name="role" value="{role: 'img', aria-label: '{file.alternative}'}" />
```

Comment:
I am not sure why do we need `aria-label: '{file.alternative}'` in case that we have `alt="{file.alternative}"`